### PR TITLE
i18n: add zh translation of "Nightly Builds"

### PR DESCRIPTION
**Pull request by zhongsp**
Translate `packages\documentation\copy\en\Nightly Builds.md` into Simplified Chinese.

### CODE REVIEW

**Comment by zhongsp** [comment]
@Kingwl , my first commit to this repo. Could you please help to review it? Thanks.

**Comment by Kingwl** [comment]
Sure.

File: packages/documentation/copy/zh/Nightly Builds.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: 每日构建
+layout: docs
+permalink: /zh/docs/handbook/nightly-builds.html
+oneline: 如何使用TypeScript的每日构建版本
+translatable: true
+---
+
+在太平洋标准时间的每日午夜，TypeScript 代码仓库中[master 分支](https://github.com/Microsoft/TypeScript/tree/master)上的代码会自动构建并发布到 npm 上。
```
Comment (Kingwl) [inline packages/documentation/copy/zh/Nightly Builds.md:9]: NIT: It's better to add UTC+8 time

> (16:00 UTC+8)

**Comment by Kingwl** [comment]
A little problem. Others look good.

File: packages/documentation/copy/zh/Nightly Builds.md (hunk 1 above)
Comment (Kingwl) [reply packages/documentation/copy/zh/Nightly Builds.md:9]: Okay never mind

**Comment by Kingwl** [comment]
LGTM

**Comment by Kingwl** [comment]
Thanks!

**Pull request merged**